appveyor.yml

```diff
@@ -0,0 +1,25 @@
+build: false
+shallow_clone: true
+environment:
+
+  global:
+    distutils_use_sdk: 1
+
+  matrix:
+    - python: "C:/Python27-x64"
+      sdkver: "v7.0"
+
+cache:
+  - '%localappdata%\pip\cache'
+
+init:
+  - ps: $Env:sdkbin = "C:\Program Files\Microsoft SDKs\Windows\" + $Env:sdkver + "\Bin"
+  - ps: $Env:sdkverpath = "C:/Program Files/Microsoft SDKs/Windows/" + $Env:sdkver + "/Setup/WindowsSdkVer.exe"
+  - ps: $Env:path = $Env:python + ";" + $Env:python + "\scripts;" + $Env:sdkbin + ";" + $Env:path
+install:
+  - ps: (new-object net.webclient).DownloadFile('https://raw.github.com/pypa/pip/master/contrib/get-pip.py', 'c:/get-pip.py')
```

Comment:
Should the copy path for the file get-pip.py be something people generally have better user rights, e.g. C:/Temp? This would be a problem in my work computer where I only have very limited rights.